README.md

```diff
@@ -5,79 +5,94 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-# Stencil Component Starter
+## How to install
 
-This is a starter project for building a standalone Web Component using Stencil.
+### NPM
 
-Stencil is also great for building entire apps. For that, use the [stencil-app-starter](https://github.com/ionic-team/stencil-app-starter) instead.
+Install dependency:
 
-# Stencil
-
-Stencil is a compiler for building fast web apps using Web Components.
+```bash
+npm i @seniorsistemas/tecnologia-webcomponents
+```
 
-Stencil combines the best concepts of the most popular frontend frameworks into a compile-time rather than run-time tool.  Stencil takes TypeScript, JSX, a tiny virtual DOM layer, efficient one-way data binding, an asynchronous rendering pipeline (similar to React Fiber), and lazy-loading out of the box, and generates 100% standards-based Web Components that run in any browser supporting the Custom Elements v1 spec.
+And import `tecnologia-webcomponents.esm.js`:
```

Comment:
I suggest include a note about legacy applications, it's needs to including only `.js` file and not `.esm.js`.

It's will be:

```html
<script type="text/javascript"...
```